src/ui/common/src/handlers/parameters/ApiKey.ts

```diff
@@ -0,0 +1,3 @@
+export type APIKeyParameter = {
```

Comment:
Should these all be in separate files, or should they all be combined into one index.ts file inside of the parameters folder?

Reply:
I like the idea of combining them. I think we can split them into `header` `path` and `query` to clearly indicate where the parameter applies